app.js

```diff
@@ -12,58 +15,60 @@ var configAuth = require('./config/auth');
 
 app.use(passport.initialize());
 app.use(passport.session());
+app.use(cookieParser());
 var connections = [];
 var GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
 var dbCOnnectionObj;
 
 // Connect to the db test
 // test commit TEST
 var port = process.env.PORT || 8080;
-MongoClient.connect("mongodb://admin:password@ds145359.mlab.com:45359/globe_trot", function(err, db) {
-  if(!err) {
-    console.log("Database connection made!");
-    dbCOnnectionObj = db;
-  }
-  else {
-      console.log('Error in connecting to database');
-  }
+MongoClient.connect("mongodb://admin:password@ds145359.mlab.com:45359/globe_trot", function (err, db) {
+    if (!err) {
+        console.log("Database connection made!");
+        dbCOnnectionObj = db;
+    }
+    else {
+        console.log('Error in connecting to database');
+    }
 });
 
-
 //adding static files
 app.use('/static', express.static('static'));
 
 //adding body parser to fetch post requests
-var bodyParser=require('body-parser');
+var bodyParser = require('body-parser');
 
 //setting view engine
 app.set('view engine', 'ejs');
-app.use(bodyParser.urlencoded({extended:true}));
+app.use(bodyParser.urlencoded({extended: true}));
 
 //session management
 app.use(sessions(
     {
-    cookieName:'session',
-    secret:'hjgwdjhnasbch2r4rcu7867hbgujgqyu287863vxqv'
+        cookieName: 'session',
+        secret: 'hjgwdjhnasbch2r4rcu7867hbgujgqyu287863vxqv'
     })
 );
 
 app.get('/', function (req, res) {
-     if(req.session){
-        req.session.reset();
+    if (req.session) {
+        // req.session.reset();
```

Comment:
dont comment this. we need to reset the session if the session already exists after user logsout